# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/apz/src/InputQueue.h
# Commit: 5e275c31e9f4
# Full Hash: 5e275c31e9f42def8437d7c8a87cf4cf7aabf0c6
# Author: Hiroyuki Ikezoe <hikezoe.birchill@mozilla.com>
# Date: 2020-11-11 09:55:06
# Regressor Bug: 1674694
# File Overlap Count: 1
# Description:
#   Bug 1674694 - Change APZEventResult::mHandledByRootApzc to Maybe<APZHandledResult>. r=botond
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D96331
# ==============================================================================

diff -r 279eb8cf0dcd -r 5e275c31e9f4 gfx/layers/apz/src/InputQueue.h
--- a/gfx/layers/apz/src/InputQueue.h	Sat Nov 07 03:43:53 2020 +0000
+++ b/gfx/layers/apz/src/InputQueue.h	Wed Nov 11 00:33:20 2020 +0000
@@ -59,7 +59,7 @@
       const RefPtr<AsyncPanZoomController>& aTarget,
       TargetConfirmationFlags aFlags, const InputData& aEvent,
       uint64_t* aOutInputBlockId,
-      Maybe<bool>* aOutputHandledByRootApzc = nullptr,
+      Maybe<APZHandledResult>* aOutputHandledResult = nullptr,
       const Maybe<nsTArray<TouchBehaviorFlags>>& aTouchBehaviors = Nothing());
   /**
    * This function should be invoked to notify the InputQueue when web content
@@ -189,7 +189,7 @@
   nsEventStatus ReceiveTouchInput(
       const RefPtr<AsyncPanZoomController>& aTarget,
       TargetConfirmationFlags aFlags, const MultiTouchInput& aEvent,
-      uint64_t* aOutInputBlockId, Maybe<bool>* aOutputHandledByRootApzc,
+      uint64_t* aOutInputBlockId, Maybe<APZHandledResult>* aOutputHandledResult,
       const Maybe<nsTArray<TouchBehaviorFlags>>& aTouchBehaviors);
   nsEventStatus ReceiveMouseInput(const RefPtr<AsyncPanZoomController>& aTarget,
                                   TargetConfirmationFlags aFlags,